spec/car_spec.rb

```diff
@@ -37,4 +37,9 @@
 
     expect(car.total_cost).to eq(54000)
   end
+
+  it 'has a color' do
```

Comment:
I think you are missing a test here. And that is why your `@color` in the initialize and your `.paint!` methods are working. The interaction pattern stated that a cars color starts out as `nil`
```
pry(main)> car.color
#=> nil

t "starts with no color" do
    car = Car.new("Ford Mustang", 1500, 36)

    expect(car.color).to eq(nil)
  end
```